src/components/LibraryItem.tsx

```diff
@@ -167,7 +167,7 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
                 <div className="LibraryItemIconWrapper">
                     <img
                         className={"LibraryItemIcon"}
-                        src={this.props.data.iconUrl}
+                        src={require("../"+this.props.data.iconUrl)}
```

Comment:
so in the context of a browser,  and not node, what does this do when transpiled?

Reply:
> so in the context of a browser, and not node, what does this do when transpiled? Maybe another way of asking this question - why is this required?

This allows webpack to get the resource and include it into the build

Reply:
How does web pack include these images in the build? I think you might find that on the Dynamo side we do some odd things to load/embed images as base64 data.

Reply:
librariejs generates the build with webpack including the bundle, index.html and the resources which are served using express for debug proposes. 
On Dynamo side we should load the build as we do in NotificationsCenter but as you mention there are odd things to handle resources. I'm debugging LibraryViewController now, and seems like we replace the resources url in the bundle according to Dynamo's current LayoutSpects.json

Reply:
Thank you for the explanation!